src/js/utils/lookupPageNameAndPageTypeDict.js

```diff
@@ -89,7 +89,10 @@ function calculatePageNameAndPageTypeDict (path) {
   let settingsPageName = 'notSet'; // Per our naming convention for pageName, this would normally be 'NotSet' but I think the value of having settingsPageName being identical to settingsPageType will save us grief in the future.
   let settingsPageType = 'notSet';
 
-  if (path.startsWith('/ballot')) {
+  if (path && (path.includes('http') || path.includes('https')) && !path.includes('wevote.us')) {
```

Comment:
For the URL `https://wevote.us/steve-garvey-politician-from-california/-/` the path is `/steve-garvey-politician-from-california/-/`

lookupPageNameAndPageTypeDictionary should NOT receive URLs, it should only receive paths  so if the path starts with `https://` or `http://` it was due to a coding mistake and it should console.error a message that includes the path that was sent in, and then it return the default values of 'notSet' for settingsPageName and settingsPageType.

If the path contains `/-/` then it is an Opinion and it should return 'OpinionSource' and 'reference'

Reply:
Fixed.